maven-java/kalix-maven-archetype-spring/README.md

```diff
@@ -0,0 +1,24 @@
+# kalix-maven-archetype-spring
+
+This archetype can be used to generate a project suitable for the development of a Service using [Kalix](https://docs.kalix.io).
+
+For the latest release see [GitHub releases](https://github.com/lightbend/kalix-jvm-sdk/releases).
+
+To kickstart a project on Linux and macOS:
+
+```shell
+mvn \
+  archetype:generate \
+  -DarchetypeGroupId=io.kalix \
+  -DarchetypeArtifactId=kalix-maven-archetype-spring
```

Comment:
I'm in doubt about the naming. 

The other archetypes start with `kalix-maven-archetype-` and then a categorisation. That makes sense because we make the choice for one or other type of component. 

While here, is an archetype for a specific Kalix SDK. 
My first thought was: `kalix-spring-maven-archetype` or `kalix-spring-boot-maven-archetype`. 

I wonder why we call it maven-archetype. It's not that we can call `mvn archetype` and have something else that is not a `maven-archetype`. 

Anyway, my current ideas are:

* `kalix-spring-boot-archetype` (my preferred)
* `kalix-spring-boot-maven-archetype` - maybe too vervose
* `kalix-maven-archetype-spring-boot` - also verbose
* `kalix-maven-archetype-spring` (currently in this PR)

WDYT?

Reply:
I agree the most accurate is `kalix-spring-boot-maven-archetype` but may be a bit too verbose. Let's save some reading time and use `kalix-spring-boot-archetype`.

Reply:
yep, let's pick `kalix-spring-boot-archetype`